{"name": "test_login_scenarios[wrong_user-wrong_pass-error]", "status": "broken", "statusDetails": {"message": "KeyError: 'label_type'", "trace": "self = <allure_pytest.listener.AllureListener object at 0x000001E409B8C440>, item = <Function test_login_scenarios[wrong_user-wrong_pass-error]>\n\n    @pytest.hookimpl(hookwrapper=True)\n    def pytest_runtest_teardown(self, item):\n        yield\n        uuid = self._cache.get(item.nodeid)\n        test_result = self.allure_logger.get_test(uuid)\n        test_result.historyId = get_history_id(\n            test_result.fullName,\n            test_result.parameters,\n            original_values=self.__get_pytest_params(item)\n        )\n>       test_result.labels.extend([Label(name=name, value=value) for name, value in allure_labels(item)])\n\nvenv\\Lib\\site-packages\\allure_pytest\\listener.py:145: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nitem = <Function test_login_scenarios[wrong_user-wrong_pass-error]>\n\n    def allure_labels(item):\n        unique_labels = dict()\n        labels = set()\n        for mark in item.iter_markers(name=ALLURE_LABEL_MARK):\n>           label_type = mark.kwargs[\"label_type\"]\nE           KeyError: 'label_type'\n\nvenv\\Lib\\site-packages\\allure_pytest\\utils.py:61: KeyError"}, "description": "Test different login scenarios", "parameters": [{"name": "username", "value": "'wrong_user'"}, {"name": "password", "value": "'wrong_pass'"}, {"name": "expected_result", "value": "'error'"}], "start": 1744384358422, "stop": 1744384365598, "uuid": "0a2cd112-b38e-4956-97e4-fee8934574ec", "historyId": "40f690503d835d0f58bfc049bf0cc74c", "testCaseId": "3b6c0d1ca0268f3d666f1f08bb6a9593", "fullName": "tests.test_login_page#test_login_scenarios"}